### DISCUSSION TITLE
Dimmer calibration

### DISCUSSION THREAD

**Discussion by bakeromso** [arendst/Tasmota#12481] [Ideas]
I remember there was talk some time ago about calibrating a dimmer from Tasmota, similarly like can be done on stock firmware for Shelly 2 dimmers. However, I couldn't find the discussion anymore and so I was wondering, does anybody know the state of this effort?
Is it already possible?
Are people working on it? 
Or do you have an idea of what might be a smart way to integrate such a feature into Tasmota?

Edit: Some background: I am currently using a Shelly 2 Dimmer, running Tasmota. The dimmer is controlling 3 dimmable LED bulbs. I suspect that their non-linear dimming behaviour might be related to the fact that they are hooked up in parallel, all to one dimmer.

**Comment by sfromis** [comment]
A lot depends on what you mean by "calibration".

**Reply by bakeromso** [reply]
Fair! So what I noticed is that I can do many steps around 0% dimmed and 100% dimmed that make a very small impact to the actual luminous intensity. However, say around 50% dimming, the same step size has a huge impact on the luminous intensity. So what I'm looking for is a way to compensate this effect, so that a step size has roughly the same impact on the luminous intensity, regardless of the current dimming percentage.
